### PR TITLE
[webui] Don't show 'create image' button for Nobody user

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

File: src/api/app/views/webui/image_templates/index.html.haml [hunk 1]
```diff
@@ -28,7 +28,7 @@
             There are currently no base image templates available for project
             = succeed '.' do
               %strong= title_or_name(project)
-  - if first_template && User.current
+  - if first_template && User.current.is_nobody?
```
Comment (Ana06) [inline src/api/app/views/webui/image_templates/index.html.haml]: shouldn't this be `if first_template && !User.current.is_nobody?` ?

**Comment by bgeuken** [comment]
It might be nicer to disable the button instead of hiding it

File: src/api/app/views/webui/image_templates/index.html.haml (hunk 1 above)
Comment (ChrisBr) [inline src/api/app/views/webui/image_templates/index.html.haml]: Yep, you're right

**Comment by ChrisBr** [comment]
Superseded by #2620, we don't need to conditional disable/hide the button when we require login for html format.

**Pull request closed without merging**